resources/logical_interconnect_group.rb

```diff
@@ -38,15 +40,57 @@ def load_lig(item)
         raise "#{up[:networkType]} #{network_name} not found" unless net.retrieve!
         up.add_network(net)
       end
-      uplink_info[:connections].each { |link| up.add_uplink(link[:bay], link[:port]) }
+      parsed_uplink_info[:connections].each { |link| up.add_uplink(link[:bay], link[:port]) }
       item.add_uplink_set(up)
     end
     item
   end
+
+  # like? function specific to the interconnectMapTemplate resource
+  def interconnect_map_like?(item, another_item)
```

Comment:
Would this be a good thing to just override in the SDK? (That way we won't have to have this sort of custom logic split out into different providers like this, and we can still use the generic create_or_update method on this and other resources)